kotori/daq/graphing/grafana/resources/grafana-target-cratedb.json

```diff
@@ -0,0 +1,10 @@
+{
+    "alias": "{{ alias }}",
+    "format": "table",
+    "resultFormat": "time_series",
+    "tags": {{ tags }},
+    "groupByTags": [],
+    "measurement": "{{ measurement }}",
+    "rawQuery": true,
+    "rawSql": "SELECT $__timeGroupAlias(time, $__interval), MEAN(fields['{{ name }}']) AS {{ alias }} FROM {{ table }} WHERE $__timeFilter(time) GROUP BY time ORDER BY time"
+}
```

Comment:
@hammerhead helped me to discover the right solution for the SQL query here, and he also told me that the `DATE_BIN()` function, which @seut recommended to use, does not yet understand Grafana's interval values.

In order to improve, we are already tracking this issue at https://github.com/crate/crate/issues/14211.

Reply:
> The `DATE_BIN()` function, which @seut recommended to use, does not yet understand Grafana's interval values.

It looks like this may change with CrateDB 5.7. Thank you, @matriv!

- https://github.com/crate/crate/issues/14211#issuecomment-1959141458